### ISSUE TITLE
Magento 1.9.2.4, PHP7, FastSimpleImport 0.7.0, Simple and config product with same images issue

### ISSUE THREAD

**Issue by Sh3rman** [avstudnitz/AvS_FastSimpleImport#310]
Hi,

Importing the simple and config product with the same images lead to image lost after reindex.
From 3 images sometimes remain 2 in the simple product and 1 in config or 3 and 0.
Looks like it's not possible to import 2 product with the same images.

I've used the import on Magento 1.9.0.1 PHP 5.5 and it worked correctly.

Does anyone had this problem or have any idea what could be?

Thank you.

**Comment by Kipperlenny** [comment]
Try not to have the same names on the images - I rename every image with "-swatch" or the color code / size.

**Comment by Sh3rman** [comment]
You mean to change the label of the image, right?
I did this:
- same images prod1.jpg, prod1_gal1.jpg, prod2_gal2.jpg for both products: simple and config,
- simple product images have no lable
- config product images have red-swatch, ... labels

Still the same result, the image is assigned to one product or the other.

I will use the swatch with images in swatch folder (not within the product images).
To fix the issue I imported the images only in the config product and all images remain in the config product.
The images are available in the frontend... it's a fix but i would like to know what could be the problem.

Thank you

**Comment by Kipperlenny** [comment]
No not the lable (only), I changed the image name!
I have a PHP skript sorting out the CSV to an Array for Import - during the image check ("do we have an image" etc.) I do a `copy($oldImage, $newImagePre . $colorCode . $size . $newImageEnd)`.
Every image has it's unique file name in the end.

**Comment by Sh3rman** [comment]
Yes, but this is not optimal to have same image multiple times just because the image can't be used by multiple products, as it did before.

In magento 1.9.0.1, PHP 5.4.45 with the same import template (same import array), same extension config works.

I have around 20 GB of images,
If each config product has 4 children and for each child product I make a image product => 
4 children + config = 5 times the images....100GB.

This is not a solution for me and it looks like a workaround just to make it work as it should and used to work.

**Comment by Kipperlenny** [comment]
> This is not a solution for me and it looks like a workaround just to make it work as it should and used to work.

Yes :-)

**Comment by Sh3rman** [comment]
I found the issue/feature.

Kipperlenny are you using "Configurable Swatches" also, right?

If you import with the Swatches off, it will work.
The images remain in the Conf and simple product as long as you have the Swatches turned off.

When you turn it back on, the image relation with the conf product will be deleted and the image will remain only in the child product.
